colossalai/booster/booster.py

```diff
@@ -64,17 +65,20 @@ def __init__(self,
             assert isinstance(
                 plugin, Plugin), f'Expected the argument plugin to be an instance of Plugin, but got {type(plugin)}.'
         self.plugin = plugin
+        logger = get_dist_logger()
```

Comment:
DON'T use this, as it relies on gpc and gpc will be deprecated soon.

Reply:
removed